src/components/Calculator.js

```diff
@@ -18,39 +18,44 @@ export default function Calculator() {
   }
 
   return (
-    <section className="">
-      <div className="row text-white text-end">
-        <Cell myClass="cell bg-dark fs-3">{total}</Cell>
-      </div>
-      <div className="row row-cols-4">
-        <Cell handleClick={(e) => handleClick(e)} myClass="cell bg-light">AC</Cell>
-        <Cell handleClick={(e) => handleClick(e)} myClass="cell bg-light">+/-</Cell>
-        <Cell handleClick={(e) => handleClick(e)} myClass="cell bg-light">%</Cell>
-        <Cell handleClick={(e) => handleClick(e)} myClass="cell bg-warning">÷</Cell>
-      </div>
-      <div className="row row-cols-4">
-        <Cell handleClick={(e) => handleClick(e)} myClass="cell bg-light">7</Cell>
-        <Cell handleClick={(e) => handleClick(e)} myClass="cell bg-light">8</Cell>
-        <Cell handleClick={(e) => handleClick(e)} myClass="cell bg-light">9</Cell>
-        <Cell handleClick={(e) => handleClick(e)} myClass="cell bg-warning">x</Cell>
-      </div>
-      <div className="row row-cols-4">
-        <Cell handleClick={(e) => handleClick(e)} myClass="cell bg-light">4</Cell>
-        <Cell handleClick={(e) => handleClick(e)} myClass="cell bg-light">5</Cell>
-        <Cell handleClick={(e) => handleClick(e)} myClass="cell bg-light">6</Cell>
-        <Cell handleClick={(e) => handleClick(e)} myClass="cell bg-warning">-</Cell>
-      </div>
-      <div className="row row-cols-4">
-        <Cell handleClick={(e) => handleClick(e)} myClass="cell bg-light">1</Cell>
-        <Cell handleClick={(e) => handleClick(e)} myClass="cell bg-light">2</Cell>
-        <Cell handleClick={(e) => handleClick(e)} myClass="cell bg-light">3</Cell>
-        <Cell handleClick={(e) => handleClick(e)} myClass="cell bg-warning">+</Cell>
-      </div>
-      <div className="row">
-        <Cell handleClick={(e) => handleClick(e)} myClass="cell bg-light col-6">0</Cell>
-        <Cell handleClick={(e) => handleClick(e)} myClass="cell bg-light col-3">.</Cell>
-        <Cell handleClick={(e) => handleClick(e)} myClass="cell bg-warning col-3">=</Cell>
-      </div>
-    </section>
+    <div className="d-flex gap-2 w-100">
+      <p className="flex-grow-1">
+        Let&apos;s do some math!
```

Comment:
- [ ] Please kindly adjust the size and alignment of this text as it is in the [wireframe](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_full_website.md), it is a best practice as a front end developer to stick to the design  as much as possible.